[ZH] Add `Client` (#8037)
ppy/osu-wiki: wiki/Help_centre/Client/zh.md

---
tags:
  - bug
  - crash
  - freeze
  - update
  - launch
  - options
  - font
  - directory
  - game
  - connection
  - bancho
  - performance
  - lag
  - glitch
  - gameplay
  - interface
---

# 客户端

*主页面：[帮助中心](/wiki/Help_centre)*
游戏客户端遇到问题了吗？你可以在下方查阅我们的用户最经常遇到的问题。

## Bug 和崩溃 {#crash}

### 我改了改设置，现在 osu! 打不开/卡住了！ {#incorrect-settings}

**大多数情况下，重置 osu! 设定为默认值可以解决此问题。**

> 跟随以下步骤来恢复 osu! 的设置：

Author: 请求修改理由？

> 跟随以下步骤来恢复 osu! 的设置：

Reviewer: 为了阅读通顺

> 1. 按住 `Shift`按键，然后打开 osu!。

Author: 我认为去除“然后”一词不能体现时间上按住按键和打开osu!的先后顺序，给读者一种“按住Shift按键就能打开osu!的感觉“。
空格问题我已经发现，感谢指正，讨论有结果后我会添加这个空格

> 1. 按住 `Shift`按键，然后打开 osu!。

Author: ```suggestion
1. 按住 `Shift` 按键，然后打开 osu!。
```

2. 持续按住 `Shift`，直到 osu! recovery 对话框出现。
3. 当 `osu! configuration` 窗口出现时，点击 `reset settings`。
4. 如果问题没有解决，依照同样的步骤回到配置窗口，这一次点击 `repair osu!`。

### 点击链接/多人游戏地图后，osu! 就卡住了！ {#broken-links}

**请确保你没有使用 Windows 兼容模式运行 osu!，且系统设置中指定过一个默认浏览器。**

> 这个问题有些时候是因为兼容模式与游戏冲突引起，也有可能只是因为 osu! 找不到用什么浏览器打开你的链接。

Reviewer: ```suggestion
这个问题有些时候是因为兼容模式与游戏冲突引起，也有可能只是因为 osu! 无法采用默认浏览器来打开你的链接。
```

#### 你的游戏正在使用兼容模式运行吗？ {#windows-compatibility}

> 在 Windows 下，若要检查你的游戏是否使用了兼容性模式，跟随如下步骤：

Reviewer: ```suggestion
在 Windows 下，若要检查你的游戏是否使用了兼容性模式，按照以下步骤做：
```

Author: 请求修改理由

Reviewer: 中文的`跟随`基本没有`跟着...去做`的意思。请求修改。

Author: 查了新华词典，确实没有，谢谢指点！

1. 打开 osu! 游戏文件夹，找到 `osu!.exe`。
2. 右键点击，然后在菜单中选择`属性`。
3. 点击`兼容性`标签。

> 4. 在名为`兼容模式`的区域，找到`以兼容模式运行这个程序`检查框。
> 5. 如果这个检查框打钩了，点一下来取消。

Reviewer: ```suggestion
4. 在名为`兼容模式`的区域，找到`以兼容模式运行这个程序`选项。
5. 如果这个选项打钩了，点一下来取消。
```

6. 如果游戏仍在运行，重启游戏。

#### 你设置默认浏览器了吗？ {#default-browser}

**绝大多数情况下，Windows 安装就会自动设置默认浏览器。**

> 跟随以下步骤检查（Windows 8及更高）：

Reviewer: ```suggestion
按照以下步骤检查（Windows 8 及更高版本）：
```

1. 打开开始菜单。
2. 在开始菜单的搜索框里输入`默认设置`，然后点击出现的设置。

> 3. 往下翻，找到`网页浏览器`，检查有没有设置成一个你已经安装的浏览器。

Reviewer: ```suggestion
3. 往下翻，找到`网页浏览器`，检查其是否设置成一个你已经安装的浏览器。
```

Author: 请求修改理由

Reviewer: 为了阅读通顺，可以保留

### 我的游戏无法完成更新！ {#cannot-update}

> **这通常是因为你的电脑所致，极少数情况下可能因为新更新有问题。**

Reviewer: ```suggestion
**这通常是因为你的电脑所致，极少数情况下，也可能是因为游戏的新版本有问题。**
```

osu! 通常会自己保持最新版本，不需要干涉，但有时情况可能有变。

我们建议你先重启一下你的电脑。不要说我们的帮助写的不走心！从电脑发明以来，重启永远能起到比想象当中更多的作用，对 osu! 来说也不例外。

确保你的"更新版本"使用的是`稳定版`。

你还可以尝试强制升级。

#### 更改更新版本 {#release-stream}

> **游戏内的设置即可进行更改。**

Reviewer: ```suggestion
**可在游戏内直接更改相关设置。**
```

Author: 请求修改理由

1. 打开 osu!。
2. 在主菜单点击 `Options`，或者按下 `Ctrl` + `O`。
3. 搜索 `release`，可以直接看到更新版本设置。
4. 确保你选择的是`稳定版`，这样才可以收到最稳定可靠的游戏版本。

> #### 强制 osu! 更新 {#force-update}
> 
> **你可以在设置里强制 osu! 进行更新**

Reviewer: ```suggestion
#### 强制更新 osu! {#force-update}

**你可以在设置里强制更新 osu!。**
```
中文语序应该是 `verb. noun. `（主谓宾）而不是 `noun. 进行一个 verb.`

Author: 认同你的修改，但是此句主语为你，谓语为强制，宾语是osu!进行更新，主谓短语作宾语，没毛病哈

1. 打开 osu!。
2. 在主菜单点击 `Options`，或者按下 `Ctrl` + `O`。
3. 搜索 `update`，可以直接看到强制更新设置。
4. 点击`启动 osu! 更新程序`。

> 你还可以在 osu! 游戏文件夹下创建一个 `help.txt` 来进行强制更新。文件不需要包含任何内容，只要在那儿，游戏下一次开启的时候就会强制进行更新了。

Reviewer: ```suggestion
你还可以在 osu! 游戏文件夹下创建一个 `help.txt` 来强制更新。文件只需要创建并处于文件夹下，不需要包含任何内容。下一次开始游戏时候，就会强制更新了。
```

Author: 请求修改理由，创建不可以表达文件的状态，如果按照您的意思，应该是”文件只需要存在并处于文件夹下。“但还是需要一个清晰的理由，为什么您认为您的修改会比我的更好。

Reviewer: 后面不是还有`不需要包含任何内容`吗... 这里主要也是为了阅读通顺。

Author: 不需要包含任何内容已经与前半句分离，结构比较不清晰，考不考虑这样的修改：
```suggestion
你还可以在 osu! 游戏文件夹下创建一个 `help.txt` 来进行强制更新。只需要创建好该文件（不需要包含任何内容）并打开游戏程序，osu! 就会自动开始更新。
```

Reviewer 2: osu! 在下一次启动时会开始强制更新

原文不是游戏目录 是安装目录

这在stable可能并不重要（虽然可以对目录结构自定义

但是lazer的游戏目录和安装目录是分离的
请尽量使用原文

Author: 但是lazer有这个功能嘛

### 我打开游戏的时候，看到关于"字体样式'的错误！ {#no-default-fonts}

> **你删除/丢失了 Windows 安装时自带的基础字体。你可能不止注意到这一个问题！**

Reviewer: ```suggestion
**你删除了 Windows 安装时自带的基础字体，或是字体已经丢失。同时，你可能不止能注意到这一个问题！**
```

> **你删除了 Windows 安装时自带的基础字体，或是字体已经丢失。同时，你可能不止能注意到这一个问题！**

Reviewer 2: ```suggestion
**你删除了 Windows 安装时自带的基础字体，或是字体已经丢失。同时，这可能不是你注意到的唯一问题。**
```
与原文不符
而且语病

> 打开 osu! 或者 osu! 更新程序的时候，如果你看到这种报错，这代表你已经删除或者不知道怎么把 Windows 安装自带的重要基础字体给丢失了。这些字体不光 osu! 会使用，其他大部分软件也会使用，所以你还可以看到其他问题的出现，例如字体错误，文字显示不出来，其他程序崩溃等等。

Reviewer: ```suggestion
打开 osu! 或者 osu! 更新程序的时候，如果你看到这种报错，这代表你已经删除字体，或者Windows 安装自带的重要基础字体不知为何而丢失了。不光 osu! 会使用这些字体，其他大部分软件也会使用，所以你还能够看到其他问题显示出来，例如字体错误，文字名无法显示，其他程序崩溃等等。
```

Author: 请求修改理由，”问题显示出来“表达比较生硬，且文字名我不知道是什么东西

Reviewer: 这里主要需要修改的是`不知道怎么把 Windows 安装自带的重要基础字体给丢失了。`这一句，基本没有`把...丢失`的说法，只有`丢失了....`

下面的文字名可以保留

Author: 感觉这样会好一点？
```suggestion
打开 osu! 或者 osu! 更新程序的时候，如果你看到这种报错，这代表你已经将字体删除，也有可能是 Windows 安装自带的重要基础字体不知为何而丢失了。不光 osu! 会使用这些字体，其他大部分软件也会使用，所以你还能够看到其他问题显现出来，例如字体错误，文字无法显示，其他程序崩溃等等。
```

> 以下乃 Windows 各版本都使用的字体，可能对你有帮助：

Reviewer: ```suggestion
以下是 Windows 各版本或 web 端都使用的字体，可能对你有帮助：
```

Author: ```suggestion
以下乃 Windows 各版本和互联网上都广泛使用的字体，可能对你有帮助：
```
查了原文是by Windows and across the Web，web应该不止指代osu!网页端吧

Reviewer: 那你加上

- [Microsoft core web fonts](https://web.archive.org/web/20020124085641/http://www.microsoft.com/typography/fontpack/default.htm) (especially Arial, Times New Roman, Trebuchet MS, and Verdana)
- [Tahoma](https://freefontsfamily.com/tahoma-font-free)
- [Windows Live Essentials](https://support.microsoft.com/en-us/help/2434419/windows-live-essentials-2011) (Segoe UI)
- [Microsoft JhengHei](https://microsoft.com/en-us/download/details.aspx?&id=12072) (微軟正黑體) (Traditional Chinese UI font)

在 Microsoft Answers 上查看[这个主题](https://answers.microsoft.com/en-us/windows/forum/windows_vista-windows_programs/font-tahoma-does-not-support-style-regular/80ad7a97-230f-41d4-9101-107a0bfa986a)来获取找回字体的相关帮助。

> ### 我的谱面丢了一部分！ {#missing-beatmaps}

Reviewer: ```suggestion
### 我丢失了一部分谱面！ {#missing-beatmaps}
```

Author: 请求修改理由

Reviewer: 理由同上。中文基本没有`把东西丢失`的说法，只有`丢失了东西`。

Author: 但是感觉这里修改完给读者一种我自己做错了什么把图给丢了的感觉，但是这里大概指代的是莫名其妙的失去了一些图？我在这里还是认为我的原文会比较好

> **请在游戏右上角看看，有没有对谱面进行分组！**

Reviewer: ```suggestion
**请看看游戏右上角有没有对谱面分组！**
```

Author: 并不是游戏右上角对谱面进行了分组，此句表达比较模糊，是在右上角可以看到，而不是右上角对谱面进行了分组。

> 如果你刚刚下载了一个新谱面，请点击 `F5` 来更新谱面列表。
> 
> 你还可以在设置中检查是否更改了谱面显示设置。

Reviewer: ```suggestion
如果你无法找到刚刚下载的新谱面，请点击 `F5` 来更新谱面列表。

你还可以在设置中检查是否更改了谱面星数显示设置。
```

> 请按照以下步骤检查是否对显示的谱面做了限制：

Reviewer: ```suggestion
请按照以下步骤检查是否限制了部分谱面的显示：
```

1. 打开 osu!。
2. 在主界面点击 `Options`，或者按下 `Ctrl` + `O`。
3. 在搜索框输入 `song select`，可以直接看到相关设置。
4. 请确保`最低显示星数`为 0，`最高显示星数`为 10+。

![](img/difficulty-display.png "选歌页面的正确值")

如果上述步骤无一生效，作为最后的解决方案，你可以尝试重建谱面数据库。**请注意此操作会导致所有谱面被标记为"未玩过"，所以将无法再依据游玩状态和日期进行搜索或分组。**

若要安全重建你的谱面数据库，遵循以下步骤：

1. 打开 osu!。
2. 在主界面点击 `Options`，或者直接按下 `Ctrl` + `O`。
3. 点击`打开 osu! 文件夹`.
4. 关闭 osu!。
5. 在新打开的文件夹里，找到 `osu!.db`。
6. 右键点击，然后点击`重命名`

> 7. 你可以随便写名字，只要不是叫 osu! 就可以。改完后按下回车。

Reviewer: ```suggestion
7. 你可以随意修改名字，只要不是 osu! 就可以。改完后按下回车。
```

Author: 请求修改理由

Reviewer: `写名字`非正式。请求修改。

8. 重新打开 osu!。

*提示：2021-05-19更新的[Stable 20210519.3](https://osu.ppy.sh/home/changelog/stable40/20210519.3)解决了这个问题。如果对你问题依然存在，请[让我们知道](https://github.com/ppy/osu-stable-issues/issues)。*

### 我的选歌界面一直在滚动！ {#songs-list-scrolling}

**这通常是因为你的输入设备出了问题。尝试拔掉所有的手柄和控制器。**

> 也有可能是因为你正在使用会改变或模拟按键绑定的软件，例如 *Xpadder* 或 *X-Mouse Button Control* 。如果你为其他游戏使用这些程序，请在玩 osu! 的时候将它们关闭。

Reviewer: ```suggestion
也有可能是因为你正在使用会改变或模拟按键绑定的软件，例如 *Xpadder* 或 *X-Mouse Button Control* 。如果你为了玩其他游戏而启用了这些程序，请在玩 osu! 的时候关闭它们。
```

Author: 请求修改理由

Reviewer: 前半句为了通顺。后半句去除被动成分。请求修改。

Author: 启用这种程序感觉是不太自然的说法
```suggestion
也有可能是因为你正在使用会改变或模拟按键绑定的软件，例如 *Xpadder* 或 *X-Mouse Button Control* 。如果你为其他游戏使用这些程序，请在玩 osu! 的时候关闭它们。
```

> 数字小键盘可以用来滚动选歌界面，因此问题也有可能因为数字小键盘。按下 `NumLock`，然后按下小键盘上的`9`，`8`，`3`或者`2`即可解决问题。

Reviewer: ```suggestion
因为数字小键盘可以用来滚动选歌界面，所以问题也有可能出自于它。按下 `NumLock`，然后按下小键盘上的 `9`、`8`、`3`、`2` 即可解决问题。
```

你还可以检查外设上是否有卡住或损坏的按键。

### 我下载了一个谱面包，但是 osu! 一直尝试"修复"文件！{#beatmap-pack-extraction}

**你需要把压缩包解压到歌曲文件夹。**

大多数谱面包的格式为 `.rar`，这意味着它们是包含很多谱面的压缩文件。你需要使用你喜欢的工具(我们推荐 [7-Zip](https://7-zip.org)) 来把压缩文件解压到你的"Songs"文件夹。

在所有的 `.osz` 文件都被解压到 `Songs` 文件夹之后，在选歌界面点击 `F5` 即可刷新歌曲缓存，把新谱面加载进入游戏。

## 游玩{#gameplay}

> ### 我的连击数/分数/准度消失了！

Reviewer: ```suggestion
### 我的连击数、分数、准度消失了！
```

Author: ```suggestion
### 我的连击数、分数和准度消失了！
```
并列关系的最后两组词中间应该使用和，或。

> ### 我的连击数、分数和准度消失了！

Reviewer 2: ```suggestion
### 我的连击数、分数和准度显示消失了！
```
连击数，分数，准度是数据
而消失的是显示它们的组件，也就是显示（display）
所以请更改

Author: 这样可以嘛
```suggestion
### 我的连击数、分数和准度不显示了！
```

**游玩时按下 `Shift` + `Tab` 会显示或隐藏这些元素，再按一次就可以重新看到了。**

如果你更改过计分板的按键绑定，那么请在`设置`->`按键设置`->`In-Game`->`Toggle Scoreboard`当中找到你绑定的按键。你也可以把 `Tab` 更改成其他按键（比如，绑定成 `Shift` + `V`）。

> ### 我的本地成绩没了！{#no-scores}
> 
> 多种原因都可能导致这种事情的发生，继续看下去吧！

Reviewer: ```suggestion
### 我的本地成绩消失了！{#no-scores}

多种原因都可能导致这种事情的发生，继续往下看！
```

Author: 请求修改理由

Reviewer: `没了`偏向于口语。

#### 你最近重装游戏了吗？{#no-scores-reinstalling}

> **重新安装游戏程序会导致本地成绩全部被清除。**
> 
> 很遗憾，因重装导致的本地成绩丢失，无法找回。
> 
> 但是，你已经提交上线的成绩仍然能够找到。只要重新下载你玩过的谱面，成绩就会重新出现。
> 
> 小贴士：如果你是 osu!supporter，你可以在 osu!direct 面板里使用`Ranked（玩过）`来筛选你玩过的谱面。你也可以在[谱面列表页](https://osu.ppy.sh/beatmapsets?played=played&s=ranked)找到你以往留下过成绩的谱面。

Reviewer: ```suggestion
**重新安装游戏程序会导致清除本地全部成绩。**

很遗憾，因重装而丢失的本地成绩，无法恢复。

但是，你仍然能够找到已经提交上线的成绩。只要重新下载你玩过的谱面，成绩就能重新出现。

小贴士：如果你是 osu!supporter，你可以在 osu!direct 面板里使用`Ranked（玩过）`来筛选你玩过的谱面。你也可以在[谱面列表页](https://osu.ppy.sh/beatmapsets?played=played&s=ranked)找到你曾经留下过成绩的谱面。
```

Author: 这里用被动语态没毛病，是成绩被清除，不是我主动清除
后面请求修改理由

Reviewer: 是你 重装游戏 导致的 清除成绩，完全是主动清除。请求修改。
第二句，导致的丢失？如果按照英文理解肯定没毛病，xxx loss 呗。但这里的主语是`丢失的成绩`，而不是`丢失`这个过程。这么修改为了让翻译更加通顺。请求修改。
第三句看你喜好，可修改可不修改。

> #### 你的游戏模式选择正确吗？{#no-scores-game-mode}

Reviewer: ```suggestion
#### 你选择了正确的游戏模式吗？{#no-scores-game-mode}
```

> **选择其他游戏模式（ osu!taiko, osu!catch 或 osu!mania）会自动隐藏其他模式的谱面成绩，只显示当前模式的成绩。**

Reviewer: ```suggestion
**选择其他游戏模式（osu!taiko、osu!catch、osu!mania）会自动隐藏其他模式的谱面成绩，只显示当前模式的成绩。**
```

Author: 顿号表并列，但游戏模式一次性只能选择一个，用或和逗号分隔没有问题

Reviewer: 也是我提到的那篇文章中出现的问题。中文中，受到了英语的影响，`A, B, C and D` 的句式变得越来越多。然而他们之间改成顿号表并列本身就是最正式的写法。

Author: 嘛，我想表达的是，顿号表并列在这里阅读起来不像是切换成其他`一个`模式，有一种同时换成这三种模式的感觉，所以觉得换成顿号会有点奇怪

Reviewer 2: 如果这样可以考虑`/`

但是以尊重原文的原则考虑，你可以将原文的这一段一块更改，如果你真的纠结。
否则应用更改就行了

Author: `/`就等同于中文中的顿号呀，还是我记错了
那这里应用了，反正中国人应该都能看得懂

> 游玩其他模式的谱面之后，默认的模式会被自动切换，因此成绩常常被隐藏而导致用户一头雾水。切换回正确的游戏模式即可。

Author: 这里用被动语态也没问题，不是我主观意识上对游戏模式进行了切换，这里不用被动语态，阅读起来真的很奇怪

> 游玩其他模式的谱面之后，默认的模式会被自动切换，因此成绩常常被隐藏而导致用户一头雾水。切换回正确的游戏模式即可。

Reviewer: 看你喜好，可修改可不修改

你可以在选歌界面的左下角找到 `Mode` 按键然后更改成你留下过成绩的模式。

#### 你最近下载过大量的谱面吗？（例如谱面包）{#no-scores-many-maps}

**成绩会在谱面导入后自动从服务器下载，若一次性导入大量谱面，下载成绩可能需要一段时间。**

> 先玩会，或者做点别的吧。游戏会在后台自动下载你的成绩。

Reviewer: ```suggestion
先玩会游戏，或者做点别的事情吧。游戏会在后台自动下载你的成绩。
```

Author: 请求修改理由

Reviewer: 更通顺。请求修改

> ### 我光标后面的这个轨迹/像烟雾的线条是什么？{#smoke}

Reviewer: ```suggestion
### 我光标后面的这个像烟雾的线条或轨迹是什么？{#smoke}
```

> **这是一个名为 `Smoke` 的功能，可以用来在游戏区域里画画。所有旁观你游戏的玩家都可以看到你的画作。**

Reviewer: ```suggestion
**这是一个名为 `拉烟 (Smoke)` 的功能，可以用来在游戏区域里画画。所有旁观游戏的玩家都可以看到你的画作。**
```

默认情况下此功能绑定 `C` 键，按住按钮就可以使用。你可以在`按键设置`的 `osu!` 这一部分更改按键绑定。

### 我怎么保存我刚刚取得的成绩回放？{#save-replay}

**在本地成绩界面戳 F2 即可。**

你漂亮的新成绩会以 `.osr` 的格式被保存在游戏路径的 /Replays/ 文件夹当中。

在你玩完一首歌之后，osu! 还会自动把所有回放保存到游戏路径下的 `/Data/r/` 隐藏文件夹当中。

> 请知悉，如果游玩后你没有马上保存回放，如此做也无法找回回放。

Reviewer: ```suggestion
请知悉，如果成绩本身不附带回放，按下 F2 也不能恢复它。
```

> ### osu!告诉我我的游戏版本过低！{#old-client}

Reviewer: ```suggestion
### osu! 提醒我：我的游戏版本过低！{#old-client}
```

> **是时候更新了！游戏的过旧版本不能提交新成绩，所以你需要点击`Options`->`常规`->`启动 osu! 更新程序`来强制更新游戏。**

Reviewer: ```suggestion
**是时候更新了！不能在过旧的游戏版本内提交新成绩，所以你需要点击 `Options`->`常规`->`启动 osu! 更新程序`来强制更新游戏。**
```

如果此举无效，你可以关闭 osu!，然后按住 `Shift` 并重新打开 osu!。如此做你会看到几个更新和修复选项，其中就有更新游戏的按钮。

> ### 我的成绩没有被提交！{#no-submission}

Reviewer: ```suggestion
### 我的成绩没有提交到服务器！{#no-submission}
```

Author: 请求修改理由

Author: 这里没有被，阅读起来也很奇怪

Reviewer: 所以后面有补充。看你喜好，可修改可不修改

嘛，这件事情可能由多种原因引起，请逐个检查：

#### 游玩的时候联网了吗？{#no-submission-no-connection}

> **如果一局游戏已经结束，你仍未联网，成绩可能无法提交。**

Reviewer: ```suggestion
**如果你在游戏结束时未能与服务器取得连接，则这个成绩可能无法提交。**
```

Author: ```suggestion
**如果一局游戏已经结束，你仍未联网，则成绩可能无法提交。**
```
请求修改理由

> 如果网络不稳定，确实很令人抓狂，但是在关闭游戏之前，游戏仍会努力尝试提交你的成绩。

Reviewer: ```suggestion
网络不稳定确实很令人抓狂，但是在你彻底关闭游戏之前，客户端仍会努力尝试提交你的成绩。
```

> **如果一局游戏已经结束，你仍未联网，则成绩可能无法提交。**

Reviewer 2: ```suggestion
**如果你在一局游戏已经结束时未联网，则成绩可能无法提交。**
```
这里联网的时机不是“整场游戏”
而是单单的游戏结束
所以不能使用仍

Author: 有理

> #### osu! 有没有被防火墙/你的杀毒软件拦截联网？{#no-submission-firewall}

Reviewer: ```suggestion
#### osu! 的联网权限有没有被你的防火墙或者杀毒软件拦截？{#no-submission-firewall}
```

**部分杀毒软件或防火墙会阻止 osu! 的网络通信，进而阻止成绩的提交。**

检查你的杀毒软件有没有允许游戏路径下的 `osu!.exe` 联网。

#### 后台是否有太多程序？{#no-submission-software}

**部分程序可能会影响成绩的提交。**

> 如果你的网络没有问题，osu! 仍然无法提交你的成绩，那么请在游玩之前关闭任何多余的程序。

Reviewer: ```suggestion
如果你的网络没有问题，而 osu! 仍然无法提交你的成绩，那么请在游玩之前关闭任何多余的程序。
```

#### 你的谱面状态正确吗？{#no-submission-beatmap-status}

**部分情况下，谱面状态会出现异常，这会导致你的成绩无法提交。**

> 在选歌界面检查屏幕左上角（例如，一张 Ranked 的谱面应该会显示向上的蓝色双箭头）。如果你的谱面没有任何图标，或图标不正确，则成绩无法上传。

Reviewer: ```suggestion
在选歌界面检查屏幕左上角（例如，一张上架 (Ranked) 谱面应该会显示向上的蓝色双箭头）。如果你的谱面没有任何图标，或图标不正确，则可能无法上传成绩。
```

![](img/correct-map-status.png "正常情况下是这个样子的")

> 游玩时选择`全球排行榜`有助于预防成绩无法提交的情况发生。如果你大部分的甚至全部谱面状态都不正常，请参阅[我的谱面丢了一部分！](/wiki/Help_centre/Client#missing-beatmaps)来重建你的谱面数据库。重建后，可能需要一段时间谱面才会恢复成正常状态。

Reviewer: ```suggestion
游玩时选择`全球排行榜`有助于预防成绩无法提交的情况发生。如果你的大部分，甚至全部谱面状态都不正常，请参阅[我丢失了一部分谱面！](/wiki/Help_centre/Client#missing-beatmaps)来重建你的谱面数据库。重建后，可能需要一段时间，谱面状态才会恢复正常。
```

> #### 成绩服务器在正常运行吗？{#no-submission-servers}
> 
> **答案几乎一定是肯定的，但是你可以检查 [osu! server status](https://status.ppy.sh) 或者 [@osustatus Twitter](https://twitter.com/osustatus) 来得知是不是我们的锅。**
> 
> 如果服务器目前不可用，那么在我们修复问题之前，请不要关闭 osu!。osu! 会在至多一个小时内一直尝试提交成绩。

Reviewer: ```suggestion
#### 记录成绩的服务器在正常运行吗？{#no-submission-servers}

**答案几乎是肯定的，但是你可以查看 [osu! 服务器状态](https://status.ppy.sh) 或者 [@osustatus 推特](https://twitter.com/osustatus) 来得知是不是我们的问题。**

如果目前服务器不可用，那么在我们修复问题之前，请不要彻底关闭 osu!。osu! 最多会在一个小时内一直尝试提交成绩。
```

Author: ```suggestion
#### 成绩服务器在正常运行吗？{#no-submission-servers}

**服务器出现故障的概率极低，但是你可以检查 [osu! server status](https://status.ppy.sh) 或者 [@osustatus Twitter](https://twitter.com/osustatus) 来得知是不是我们的锅。**

如果服务器目前不可用，那么在我们修复问题之前，请不要关闭 osu!。osu! 会在至多一个小时内一直尝试提交成绩。
```
此句有语病，几乎是肯定的相互矛盾，几乎一定是肯定的问题比较轻微，但是还是有问题
其他部分请求修改理由

Reviewer: 一定和肯定相互矛盾。很明显的病句。参考：https://zhidao.baidu.com/question/373679921653091084.html 必须修改。

最上面的标题可不修改。
`是...的锅`很明显一种网络用语，非常不正式。请求修改。
其余部分均为了更加通顺。

Author: 这样可以嘛
```suggestion
#### 成绩服务器在正常运行吗？{#no-submission-servers}

**服务器出现故障的概率极低，但是你可以检查 [osu! server status](https://status.ppy.sh) 或者 [@osustatus Twitter](https://twitter.com/osustatus) 来得知是不是我们的问题。**

如果服务器目前不可用，那么在我们修复问题之前，请不要关闭 osu!。osu! 会在至多一个小时内一直尝试提交成绩。
```

Reviewer 2: 其实这里有个小问题
ppy已经将成绩上传超时时间改了
所以应该不是至多一小时
不过这要求同时改英语原文

看你怎么想

Author: 等英文正式改了再改中文吧 我就是做个翻译 原文对不对 正不正确不是我要考证的 我也无从考证（

> ### 你可以帮我上传一份成绩吗？{#upload-replay}

Reviewer: ```suggestion
### 你们可以帮我上传成绩吗？{#upload-replay}
```

Author: 请求修改理由

**很遗憾，我们不能。**

> 无论因为任何原因，如果成绩提交最终失败，那么只能祝愿你能够让奇迹可控化，加油，抱歉。

Reviewer: ```suggestion
如果你因为技术或其他原因未能上传成绩，你最好寄托于复现你的操作并再次获得好成绩。我们深感抱歉！
```

????????? wtf 这啥啊？？

Author: 我们又不是在写论文，百科全书，或者法律条文，语言风格没必要这么认真吧
请求修改理由，原文虽然是If you were denied a score due to technical reasons or anything else，但是只要表达出来因为任何原因都不行就是准确的表达。奇迹可控化是对玩家取得好成绩的祝愿，因为好成绩可遇不可求，有奇迹的意味。此句”寄托于复现你的操作“表达也有问题，寄托什么？把什么寄托？

Reviewer: 参考原文：If you were denied a score due to technical reasons or anything else, the best you can hope for is to repeat your display of skill and achieve it again. Sorry!

首先，osu!wiki，取名于 [WIKI 维基百科](https://zh.wikipedia.org/wiki/Wikipedia:%E9%A6%96%E9%A1%B5)。你要觉得这不算百科，可以随意用词，那我是无法苟同的。

其次，`奇迹可控化是对玩家取得好成绩的祝愿`，根据我的中文理解力，我无法理解这一`...化`到底表达的是什么意思。诸如`...化`明显翻译于英文词尾：`-zation` / `-sation` / `-able`。明显的中式英文。

并且，原文并没有`miracle controllable / 奇迹可控化`一意，只有`you can hope for / 你可以希望 / 你寄托于`一意。其他词不达意的地方（repeat your display 的 `repeat` 和 achieve it again 的 `again` 表示的`重复`一意未能体现，等等）我就不提了。

再者，`due to technical reasons` 是这 `anything else` 中很重要的一环，不可或缺。如果不翻译会丢失原意。

最后，`寄托于复现你的操作` 对应于 `hope for is to repeat your display`。这里使用直译本身就合理通顺，实在没必要自顾自地去捏造一个`奇迹可控化`，来显得很高级。如果有其他问题，我的理由就是：请对应原文。

必须修改。

Author: 这样能行嘛
```suggestion
如果因为技术或任何其他原因，导致你的成绩没能成功上传，那么你只能寄希望于复现你的操作并再次获得好成绩了。我们深感抱歉！
```

> ### 物件好像和音乐八竿子打不着！{#offsync-notes}

Reviewer: ```suggestion
### 物件好像和音乐不同步！{#offsync-notes}
```

八竿子打不着，和风马牛不相及差不多，是表示完全没有关系 (not revelant)。
而这里物件和音乐只是一个快一个慢，如果用这种俗语则大错特错。

Author: 有理

**这可能是因为你电脑上的音频软件导致了问题，例如 *Razer Surround Audio* 。也有可能是因为你的`全局偏移量`设置有误。**

> 尝试关闭电脑上会对音频信号作处理的软件。如果没有效果，你可以尝试更改`全局偏移量`。

Reviewer: ```suggestion
尝试关闭电脑上会处理音频信号的软件。如果没有效果，你可以尝试更改`全局偏移量`。
```

每一套硬件都会有独特的`全局偏移量`值，这决定了你听到音乐的时机会早一点还是晚一点。

默认值适用于大多数人，但是如果不适合你，你可以跟随下列步骤来进行更改：

1. 打开 osu!。
2. 在主界面点击 `Options`，或按下 `Crtl` + `O`。
3. 在搜索框输入 `offset`，这样就可以看到全局偏移量设置。

> 4. 使用滑条来更改之。你也可以尝试使用 [Offset 设置向导](/wiki/Client/Options/Offset_Wizard)来辅助你调整。

Reviewer: ```suggestion
4. 调整滑块来更改全局偏移量。你也可以尝试使用 [Offset 设置向导](/wiki/Client/Options/Offset_Wizard)来辅助你调整。
```

Author: 请求修改理由，之就指代了全局偏移量啊

Reviewer: 原文：Use the slider to adjust your universal offset

可以将`之`修改为`它`。即使在现代比较正式的中文里，也很少用`之`代替`it`。这是古代文言文的用法。

我更建议修改`滑条`，它很容易对上的是游戏术语滑条。会引起误会。

Author: ```suggestion
4. 使用滑块来更改它。你也可以尝试使用 [Offset 设置向导](/wiki/Client/Options/Offset_Wizard)来辅助你调整。
```

如果一切都不奏效，你可以尝试更新声卡或主板音频驱动。

> ### 我的游戏看起来好像有问题，画面好像被撕裂！{#screen-tearing}

Reviewer: ```suggestion
### 我的游戏看起来好像有问题，出现了撕裂画面！{#screen-tearing}
```

> **这是由名为[画面撕裂](https://en.wikipedia.org/wiki/Screen_tearing)的现象造成的，把`帧率限制`设置更改为`垂直同步`即可解决问题。**

Reviewer 2: ```suggestion
**这是由名为[画面撕裂](https://zh.wikipedia.org/wiki/%E7%95%AB%E9%9D%A2%E6%92%95%E8%A3%82)的现象造成的，把`帧率限制`设置更改为`垂直同步`即可解决问题。**
```
你想的话这个也可以，但是有中文引用就不要用英文
https://baike.baidu.com/item/%E7%94%BB%E9%9D%A2%E6%92%95%E8%A3%82/15626593

Author: ```suggestion
**这是由名为[画面撕裂](https://baike.baidu.com/item/画面撕裂/15626593)的现象造成的，把`帧率限制`设置更改为`垂直同步`即可解决问题。**
```

这种现象并非软件或硬件出现问题（虽然看起来有点诡异），单纯是因为游戏的刷新率与你的显示器不相同。

你也可以尝试使用窗口化或全屏无边框模式游玩，这会让 osu! 跟随系统调度垂直同步。

> 很遗憾，在窗口化或全屏无边框模式下打开垂直同步会导致游戏增加*些许的*输入延迟。对大多数人来讲，这不应该有很大的影响。

Reviewer: ```suggestion
很遗憾的是，在窗口化或全屏无边框模式下启用垂直同步，会增加*些许的*游戏输入延迟。对大多数人来讲，影响不会很大。
```

Author: 请求修改理由

Reviewer: 为了通顺。建议修改。

较新的 NVidia 显卡可以在 NVidia 控制面板中把全局刷新率设置为'快'。这也能解决问题。

## 在线功能{#online-features}

> ### 我无法登录/游戏根本无法联网！{#cannot-sign-in}

Reviewer: ```suggestion
### 我无法登录，或是游戏根本无法联网！{#cannot-sign-in}
```

> **请确保你电脑上的防火墙/杀毒软件没有禁止 osu! 进行网络通信。**

Reviewer: ```suggestion
**请确保你电脑上的防火墙或者杀毒软件没有禁止 osu! 的网络通信权限。**
```

以下为常见杀毒软件允许程序联网的方式：

#### 在 Windows 防火墙当中允许程序联网{#firewall-whitelist-windows}

**在大多数的 Windows 系统中，Windows 防火墙是默认的防火墙软件。**

若要确保 osu! 可以联网，需要把 osu! 加入 Windows 防火墙的白名单。

1. 打开开始菜单。
2. 在开始菜单的搜索框当中，输入`允许应用`。
3. 你应该会看到名为`允许应用通过 Windows 防火墙`的设置面板项。点它。
4. 一个有很多程序列成表，程序旁边有两个复选框的面板会打开。随便点击一个程序的名字，然后按下`o`按钮。这会让列表滚动到以"o"开头的程序那里。

> 5. 找找看 `osu!` 或 `osu!.exe`。

Reviewer 2: ```suggestion
5. 找找看是否有 `osu!` 或 `osu!.exe`。
```
找找看是动作，是否是目的
原句有点缺少成分

Author: 其实想想，这里基本是一定有的吧，而且目的是找到它所在的位置，不是有没有，感觉没有必要加个是否？

Reviewer 2: ```suggestion
5. 寻找 `osu!` 或 `osu!.exe`。
```

Author: 那这样也没有意义呀，找找看比寻找要生动，能给读者带来愉悦的阅读体验，而且意思也一样啊

6. 如果这两个都没有，请点击面板底部的`允许其他应用`，然后搜索你的 osu! 安装路径，在里面选择 `osu!.exe`。
7. 请确保`专用`和`公用`复选框都打上了勾，然后点击确认保存，打开游戏重新尝试。

#### 允许程序通过 AVG 防火墙 {#firewall-whitelist-avg}

请参阅 AVG 的[官方知识库文章](https://support.avg.com/SupportArticleView?l=en&urlName=AVG-Antivirus-Enhanced-Firewall-Application-Rules&supportType=home)来了解如何操作。

> #### 允许程序通过 Kaspersky 互联网安全 {#firewall-whitelist-kaspersky}
> 
> 请参阅 Kaspersky 的[官方支持文章](https://support.kaspersky.com/15163#block2)来了解如何把程序加入信任名单。

Reviewer: ```suggestion
#### 允许程序通过卡巴斯基 (Kaspersky) 互联网安全 {#firewall-whitelist-kaspersky}

请参阅卡巴斯基的[官方支持文章](https://support.kaspersky.com/15163#block2)来了解如何把程序加入信任名单。
```

> #### 允许程序通过 McAfee {#firewall-whitelist-mcafee}
> 
> 请参阅 McAfee 的[官方知识库文章](https://service.mcafee.com/?articleId=TS100813&page=shell&shell=article-view)。

Reviewer: ```suggestion
#### 允许程序通过迈克菲 (McAfee) {#firewall-whitelist-mcafee}

请参阅迈克菲的[官方知识库文章](https://service.mcafee.com/?articleId=TS100813&page=shell&shell=article-view)。
```

### 我已经允许 osu! 通过防火墙了，但是我仍然没办法连接游戏！ {#sign-in-unknown-cause}

**我们需要更多信息才能帮助你解决问题。**

如果你仍然不能连接上游戏服务器，请发邮件给 [accounts@ppy.sh](mailto:accounts@ppy.sh) 来申请帮助。请提供以下信息来帮助我们快速解决你的问题：

- 使用 `tracert` 命令，获取对 `cho.ppy.sh`，`osu.ppy.sh` 和 `m1.ppy.sh` 的路由追踪日志（可参照下方的指示）。
- 游戏安装目录，`Logs` 文件夹内的 `network.log`。

#### 进行路由追踪来排除网络问题 {#traceroute}

**路由追踪可以让我们检测你与游戏服务器中间到底出了什么问题。**

你可以在命令提示符中使用 `tracert` 命令来进行路由追踪。

> 如果要打开命令提示符，请按住键盘上的 `Windows` 键，然后点击 `R`。然后在弹出的窗口中输入 `cmd` ，然后按下 `确认`。详细信息请参阅[这篇 Windows 知识库文章](https://support.microsoft.com/en-us/topic/how-to-use-tracert-to-troubleshoot-tcp-ip-problems-in-windows-e643d72b-2f4f-cdd6-09a0-fd2989c7ca8e)。

Reviewer 2: ```suggestion
如果要打开命令提示符，请按住键盘上的 `Windows` 键，然后点击 `R`。然后在弹出的窗口中输入 `cmd` ，然后按下 `确定`。详细信息请参阅[这篇 Windows 知识库文章](https://support.microsoft.com/zh-cn/topic/%E5%A6%82%E4%BD%95%E4%BD%BF%E7%94%A8-tracert-%E8%A7%A3%E5%86%B3-windows-%E4%B8%AD%E7%9A%84-tcp-ip-%E9%97%AE%E9%A2%98-e643d72b-2f4f-cdd6-09a0-fd2989c7ca8e)。
```

在 windows 11 下 内部版本 22623.891 中，运行的按钮为 `确定`

随后，你就可以把指令输入进去了。举个例子，`tracert cho.ppy.sh` 会对你电脑与游戏服务器的通信进行追踪。这可能需要一分钟或者几分钟，具体依据你的网络情况而定。

我们需要对 `cho.ppy.sh`, `osu.ppy.sh`，和 `m1.ppy.sh` 的路由追踪日志。

如果要复制追踪结果，点击运行结果的底部，向上拖动，结果都变白之后即可右键复制。

### 在线游玩时，我收到了服务器端的错误！ {#server-side-error}

**游戏服务器可能正面临问题。**

请看看 [osu! 服务器状态页面](https://status.ppy.sh/ "osu! server status")来确定是不是服务器出现了故障。

**这也可能因为你使用了共享网络连接，代理服务器或者 VPN。**

尝试换一个代理服务器或 VPN 服务提供商，如果不奏效，请发邮件给 [accounts@ppy.sh](mailto:accounts@ppy.sh) 来请求帮助。

如果你没有用过那些东西，那么请直接请求帮助，你的连接可能被我们误封禁了。

Tor 网络或通过 Tor 的连接将无法连接大部分游戏服务器。虽然我们对于 Tor 的使用不持任何立场，但是我们如此做只是为了阻止坏人使用 Tor 干坏事。

### 我赞助了 osu!，但是我的用户名不是黄的！ {#white-username}

**在游戏客户端当中，你自己的用户名永远是白色。其他玩家可以看到你闪闪发光的黄色名字！**

这可以帮助你区分你自己和其他人的消息，目前无法禁用这个功能。

## 性能 {#performance}

### 我的电脑运行 osu! 会过热，怎么让它冷下来？ {#overheating}

**最简单的方式就是改变你的帧数限制设置。**

> 若要检查你目前的帧数限制，请参阅如下步骤：

Reviewer 2: ```suggestion
要检查你目前的帧数限制，请参阅如下步骤：
```
原文没有 if，这里不需要询问语气

Author: 但是我感觉改完去掉这一个字真的读起来很诡异很诡异...这里是我的主观想法，但是我觉得加上也没啥

1. 打开 osu!。
2. 在主页点击 `Options`，或者按下 `Ctrl` + `O`。
3. 直接输入 `frame` 来看到你目前的帧数限制设置。

如果你的设置是`无限制`，把设置更改为其他三个可能会有明显的效果。

> `垂直同步`会让游戏运行在你的显示器帧率下，而 `Power Saving` 会让游戏运行在显示器的两倍帧率下，`Optimal` 会让游戏运行在显示器的八倍帧率下。

Reviewer 2: ```suggestion
`垂直同步`会让游戏运行在你的显示器帧率下，而 `Power Saving` 会让游戏运行在显示器的两倍帧率下，`Optimal` 会让游戏运行在显示器的八倍帧率下，最高 960Hz。
```
漏了

如果你的电脑会过热，我们建议选择`垂直同步`。

你还可以了解一些控制处理器和显卡风扇转速的程序。

> ### 玩 osu! 的时候，我的电脑自动重启！ {#reboot-overheating}

Reviewer: ```suggestion
### 我的电脑会在玩 osu! 的时候自动重启！ {#reboot-overheating}
```

> **多数时候，这是因为你的电脑一个或多个部件过热了，大概率是你的显卡或者处理器。**

Reviewer: ```suggestion
**多数时候，这是因为你的电脑数个部件过热了，大概率是你的显卡或者处理器。**
```

Author: 数个大多数情况下只能表多个，无法表达”一个或多个“的意思。

Reviewer: 可以不修改

如果要搞清楚原因，我们建议你使用检测显卡和处理器温度的程序，并且在游戏运行的时候实时关注温度值。一个很好的选择是 [Speccy](https://www.ccleaner.com/speccy)。

如果你看到你的处理器和或显卡已经超过了 90°C，你的电脑就已经非常热了，并且非常接近自动重启的上限。

### 我的电脑没有过热，但是游戏仍然会导致我的电脑重启！ {#reboot-faulty-device}

**过热后的第二大原因就是声卡问题，第三大原因是你的外设出现问题。**

旧一些的声卡驱动或软件可能会导致 osu! 出现问题，但是对大多数人来讲这不属于大问题。尝试更新你的声卡或主板驱动。

你还可以试一下拔除连接到你电脑的一些 USB 外设，尤其是那些处理声音信号的设备。耳机，USB 声卡或它们连接到的接口一旦坏掉，可能会导致问题。

> ### 还是重启！ {#reboot-nothing-helps}

Reviewer: ```suggestion
### 试过以上方法后还是重启！ {#reboot-nothing-helps}
```

如果你已经试过所有方式，问题仍然出现，请联系 [accounts@ppy.sh](mailto:accounts@ppy.sh) 来请求帮助。请附上有用的信息，比如你的电脑配置，型号（如果适用）和你认为可能有助于解决问题的其他信息。

> ### 玩 osu!mania 的时候，我不能一次性按下 n 个按钮！ {#keyboard-ghosting}

Reviewer 2: ```suggestion
### 玩 osu!mania 的时候，我不能一次性按下多个键！ {#keyboard-ghosting}
```
首先和原文不一致
接着如果是这种标题不利于真正需要它的玩家，没有人会这样搜索
n个按键在中文的语境中应该是不存在的，所以也不应该使用。
最后就算要使用这种标题，也没有叫明白n到底是什么
故请修改

Author: 原文是more than a few keys，多个键也不贴合英文原文，最好的说法应该是`多于几个键`，那`多于几个键`这种表达我认为还不如直接写`多于 n 个键`。况且多本质上和n也没有区别啊，都是设了一个未知数，一定要写上n>2才可以吗
而且多在这里容易引起歧义呀，2个也叫多，99个也叫多，n至少能代表到一个精确的值，多个就把这数量彻底模糊化了，我感觉更不利于玩家搜索与查找
我认为这句话的翻译确实有待商榷，但是您的编辑我不敢苟同

**这是因为大部分键盘都存在的*按键冲突*问题。**

按键冲突是个有一点复杂的问题，本质是你的键盘制作工艺上的缺陷，导致它不能发送一次性按下多个键的信号。

> 大部分的新键盘可以一次性按下**6**个按键，顺序和按什么不限。

Reviewer 2: ```suggestion
大部分的新键盘可以一次性按下**6**个按键，不管按下的是什么键。
```
```suggestion
大部分的新键盘可以一次性按下**6**个按键，就算是随便按下的。
```
按键 顺序 都为两个字的词，很容易被认为是一组连续阅读而导致语义不通。
请避免这种情况

大部分的新键盘可以一次性按下**6**个按键，顺序 | 和按什么不限。

> 你可以用 [drakeirving 的键盘冲突小工具](https://drakeirving.github.io/MultiKeyDisplay)来检查自己的键盘一次性可以正常按下多少个键。

Reviewer 2: ```suggestion
你可以用 [drakeirving 的键盘冲突小工具](https://drakeirving.github.io/MultiKeyDisplay)来检查自己的键盘可以一次性正常按下多少个键。
```
在正常语序中可以应当提前，原文也是如此

Author: 但是意义呢 我感觉我的表达也没问题呀 一定要精确到每一个字符和每一条语法吗 而且本来我这也没有严重的影响阅读语法错误啊 那这么说您直接自己翻译这一整篇就可以了呀 您想用什么语言风格就用什么语言风格
我感觉我这一句没有错误 不需要改

虽然 osu!mania 的 Ranking Criteria 已经规定谱面不能出现同时按下6个键以上的配置，但如果你的键盘比较老旧，按键冲突的问题可能还是会影响到你。

> 治标治本的解决方案只有购买一个能同时处理更多按键的新键盘。如果那样的话，你可能可以关注*全键无冲*的键盘。这种键盘基本不会有按键冲突的问题，但是一般比较昂贵。

Reviewer 2: ```suggestion
治标治本的解决方案只有购买一个能同时处理更多按键的新键盘。如果那样的话，你可以关注*全键无冲*的键盘。这种键盘基本不会有按键冲突的问题，但是一般比较昂贵。
```
```suggestion
治标治本的解决方案只有购买一个能同时处理更多按键的新键盘。如果那样的话，你可能需要关注*全键无冲*的键盘。这种键盘基本不会有按键冲突的问题，但是一般比较昂贵。
```

原文为 `may wish`。但是在中文语境中，可能可以关注三个词的组合并不合适，你也可以考虑其他的修改
但是不能保留

Author: 一会贴近原文，一会不能按原文来，咱这是...啥情况
我认为一篇翻译文章值得进行修改的只有两种情况：
1. 读者读下去的当下理解错误，想了想才反应过来正确的理解方式
2. 直接性的表达错误或不地道的表达方式导致阅读体验不好
这种吹毛求疵的更改咱能不能别整了，浪费彼此时间

Reviewer 2: 在能符合正常中文交流语境中，尽量贴近原文
但是在原文翻译为中文时无法正常符合正常语境则可以适当修改和润色，这样读起来也舒服

### 玩 osu! 的时候，我的光标疯跑！ {#crazy-cursor}

> **最常见的原因是你的游戏内鼠标灵敏度设置的太高了！**

Reviewer 2: ```suggestion
**最常见的原因是你游戏内的鼠标灵敏度设置的太高了！**
```
The same as Google translate
你游戏内 vs 你的游戏内

游戏内的鼠标灵敏度 vs 游戏内鼠标灵敏度

乱跳，诡异的鼠标移动是最典型的例子，尤其是当你使用数位板的时候。

大部分情况下，把游戏内的灵敏度更改为 1.0x（默认设置）会解决这个问题，但是可能会导致你的鼠标移动比较缓慢。我们建议你打开游戏中的 `Raw Input` 来让 osu! 直接通过设备驱动来调整灵敏度。

如果你使用鼠标，你还可以尝试更改你的鼠标 *DPI* 设置。更高的 DPI 设置能让游戏收到更准确的定位，同时可能会平滑掉迟缓的感觉。

我们还建议你检查自己的外设是否足够的干净。如果有头发或者灰尘挡住鼠标的激光传感器，可能会导致鼠标移动很怪。

> #### 启用 Raw Input {#raw-input}
> 
> **Raw input 会让游戏直接与你的外设通信，获得更加直接且未经处理的移动数据，这通常会让设备输出的精度提升。**
> 
> 若要打开 Raw Input，请参阅以下步骤：
> 
> 1. 打开 osu!。
> 2. 点击主页的 `Options`，或者按下 `Ctrl` + `O`。
> 3. 直接输入 `raw`，即可跳转到对应的设置区域。
> 4. 点击 `Raw input` 来将它打开。

Reviewer: ```suggestion
#### 启用原始输入 (Raw Input) {#raw-input}

**原始输入 (Raw Input) 会让游戏直接与你的外设通信，获得更加直接且未经处理的移动数据，这通常会让设备输出的精度提升。**

若要打开原始输入 (Raw Input)，请参阅以下步骤：

1. 打开 osu!。
2. 点击主页的 `Options`，或者按下 `Ctrl` + `O`。
3. 直接输入 `raw`，即可跳转到对应的设置区域。
4. 点击 `Raw input` 来将它打开。
```

### 我使用数位板，我感觉我的光标移动太慢了！ {#slow-cursor}

**你可以设置更小的数位板映射区域来解决问题。**

在大多数 Wacom 数位板上，你可以在 `Wacom 数位板属性`的`映射`标签页中完成操作。

> ![Wacom Tablet Properties screen showing Mapping options](img/tablet-area.png "Interface for changing tablet area on Wacom devices")

Reviewer: ```suggestion
![Wacom 数位板属性屏幕显示映射选项](img/tablet-area.png "Interface for changing tablet area on Wacom devices")
```

Reviewer 3: the tooltip has to be translated as well

Author: ```suggestion
![Wacom 数位板属性屏幕显示映射选项](img/tablet-area.png "Wacom 设备更改数位板映射区域的界面")
```

> 你需要咨询你的设备生产商来了解如何为你的设备更改映射区域。

Reviewer: ```suggestion
你需要咨询你的设备生产商来了解如何修改你的设备的映射区域。
```

> 所有 osu! 品牌的数位板均支持 Raw Input，可以直接使用游戏中的鼠标灵敏度来轻松调整光标灵敏度。

Reviewer: ```suggestion
所有 osu! 品牌的数位板均支持原始输入 (Raw Input)，可以直接使用游戏中的鼠标灵敏度来轻松调整光标灵敏度。
```

### 玩游戏的时候，我的电脑出现卡顿！ {#lag-spikes}

**游戏出现卡顿有多种原因，可能是因为软件或过期的驱动，也可能只是显示设置调整过高。**

#### 玩 osu! 的时候，你的后台有很多程序吗? {#lag-too-many-programs}

**部分程序可能会与 osu! 争抢系统资源，造成游戏体验不佳。**

尝试关闭一部分或者全部玩游戏时不需要的程序。

#### 你的电脑安全吗？ {#lag-infected-pc}

**病毒，后门程序，恶意程序或其他你可能不想要的程序可能会导致严重的性能问题，哪怕是很高级的电脑。**

保证电脑安全可能会改善问题。尝试在电脑上运行病毒查杀来解决这些问题。

#### 你的电脑用多久了？ {#lag-old-pc}

**非常久的电脑可能使用更旧的 Intel 集显，导致无法运行游戏，就算设置已经全部改成最低。**

解决问题的唯一方法恐怕只有升级电脑了。

#### 你的驱动是最新版本吗？ {#lag-old-driver}

**过期的驱动可能导致严重的性能问题。如果可以，请一直保持驱动为最新状态。**

显卡驱动对 osu! 比较重要，声卡驱动次之。

#### 你使用一个 USB 集线器连接了很多 USB 设备吗？ {#lag-faulty-usb}

**只使用一个插孔并用集线器连接太多设备可能会导致输入延迟过高，这通常很难以发现，但是会对游戏造成较大影响。**

一个简单的解决办法是：拔掉集线器，如果问题解决，你就知道问题所在啦！

#### 你正在使用窗口化或全屏无边框模式运行 osu! 吗？ {#lag-windowed}

**窗口化和全屏无边框均会带来*可观*的输入延迟。若要保证游戏体验，请使用全屏模式。**

#### 你的游戏只是卡顿一小会吗？ {#lag-background-process}

**有时候，电脑上的后台程序可能会一次性处理过多数据，这可能会让机器暂时力不从心。**

在玩游戏之前关闭不必要的后台程序通常可以解决问题。P2P 和种子下载程序通常影响最大，尤其是当它与 osu! 安装在同一块硬盘上的时候。

#### 游戏最近更新了吗？ {#lag-after-update}

**有些时候，更新客户端后可能会导致客户端莫名卡顿。**

如果你有很多谱面，在怪罪更新之前，请等待 10-15 分钟。osu! 可能正在后台重新计算谱面难度。

如果等待后无法解决问题，重启可能会有帮助。

> **如果游玩时你仍然会遇到卡顿停顿问题，请参阅[排除性能问题的指导](/wiki/Performance_troubleshooting)来获知更多问题和可能的解决方案。**

Reviewer 2: ```suggestion
**如果游玩时你仍然会遇到卡顿停顿问题，请参阅[osu! 性能问题检查](/wiki/Performance_troubleshooting)来获知更多问题和可能的解决方案。**
```
当链接其他已经翻译的文章时，请直接使用翻译后文章的标题
